CSS Assignment 2/website.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <!--header starts-->
+  <head>
+    <title>
+      Information Page
+    </title>
+    <meta name = "author" content = "Akshita Dixit">
+    <link type="text/css" href="style.css" rel="stylesheet"/>

```

Comment:
CSS files should be placed under a dedicated folder (like `styles`) instead of keeping them at the same level as the HTML files.